AfricanReligions/decisions/afr_decisions.txt

```diff
@@ -750,4 +750,64 @@ title_decisions = {
 			factor = 0
 		}
 	}
-}
\ No newline at end of file
+	
+	pick_religious_tradition = {
+		only_playable = yes
+		is_high_prio = yes
+		ai_check_interval = 12
+		potential = {
+			has_dlc = "The Old Gods"
+			is_playable = yes
+			is_adult = yes
+			prisoner = no
+			OR = {
+				religion = a_tag_roog
+				religion = a_tag_roog_reformed
+			}
+			NOT = {
+				has_alternate_start_parameter = { key = religion_names value = random }
+			}
+			NOR = {
+				if = {
+					limit = {
+						ai = yes
+						has_trait = follower_of_serer
+					}
+				}
+				if = {
+					limit = {
+						ai = yes
+						has_trait = follower_of_dogon
+					}
+				}
+				if = {
+					limit = {
+						ai = yes
+						has_trait = follower_of_malinke
+					}
+				}
+			}
+		}
+		allow = {
+			piety = 25
+			prisoner = no
+		}
+		effect = {
+			piety = -25
+			custom_tooltip = {
+				text = pick_religious_tradition_tooltip
+				character_event = { id = GROUPS.1000 }
+			}
+		}
+		revoke_allowed = {
+			always = no
+		}
+		ai_will_do = {
+			factor = 0.5
+			modifier = {
+				factor = 0
+				has_minor_deity_trigger = yes
+			}
```

Comment:
We probably just want the AI to do this immediately regardless, if the decision is available.  `has_minor_deity_trigger` only includes patron deity modifiers from other pagan religions, so it will never be true for our religions unless we modify the trigger anyway (which we should, but only for the patron deity modifiers, not these religion branches).